tests/test_file_watcher.py

```diff
@@ -102,7 +102,7 @@ def tearDown(self) -> Generator:
         self.assertEqual(len(TestFileWatcher._active_watchers), 0)
         # Restore original project data.
         window = sublime.active_window()
-        window.set_project_data(None)
+        window.set_project_data({})
```

Comment:
is this intentional?
what is the difference between None and {}?

Reply:
Fix for type issue. According to types it doesn't accept `None`.
I don't think there are differences in practice. The tests still pass at least.